src/lesson6/Animals.java

```diff
@@ -0,0 +1,107 @@
+package lesson6;
+
+import java.util.Objects;
+
+public class Animals {
```

Comment:
```suggestion
public class Animal {
```
класс подразумевает описание одного животного, когда будет создавать от него объекты